test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Caching/CachingTestContext.cs

```diff
@@ -251,9 +251,28 @@ private void InitializeFiles()
             ProjectPath = Path.Combine(WorkingPath, "project.csproj");
 
             OutputPackagesPath = Path.Combine(WorkingPath, "packages");
+            CreateNuGetConfig(WorkingPath);
+
             Directory.CreateDirectory(OutputPackagesPath);
         }
 
+        private void CreateNuGetConfig(string workingDirectory)
+        {
+            string nugetConfigContent =
+                @"<?xml version='1.0' encoding='utf-8'?>
+<configuration>
+  <packageSources>
+    <clear />
+  </packageSources>
+  <packageSourceMapping>
+    <clear />
+  </packageSourceMapping>
+</configuration>";
+            var fileName = "nuget.config";
```

Comment:
Does it make sense to use `NuGet.Configuration.Settings.DefaultSettingsFileName` instead?

Reply:
Sure.